assets/js/components/surveys/SurveyQuestionSingleSelectChoice.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * SurveyQuestionSingleSelectChoice component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Radio from '../Radio';
+import { TextField, Input } from '../../material-components';
+
+const MAXIMUM_CHARACTER_LIMIT = 100;
+
+/* eslint-disable camelcase */
```

Comment:
As mentioned elsewhere, we tend to do this per-line to be extra deliberate. It's more annoying to write but ensures every instance is intentional 😅

Reply:
Done!